### ISSUE TITLE
filters not working in online demo

### ISSUE THREAD

**Issue by zjk261** [ruffle-rs/ruffle#18307]
### Describe the bug

![Image](https://github.com/user-attachments/assets/3e073642-4352-44cd-9781-474df218dc17)
![Image](https://github.com/user-attachments/assets/4f99f279-c269-4df6-ba31-6ff6e2c8eb7d)

### Expected behavior

it should work

### Content Location

[551349_blosics2_levelpack.zip](https://github.com/user-attachments/files/17411705/551349_blosics2_levelpack.zip)
[BulletHeaven2.zip](https://github.com/ruffle-rs/ruffle/files/12115384/BulletHeaven2.zip)

### Affected platform

Online demo

### Operating system

Windows 10

### Browser

Google Chrome 130.0.6723.59

### Additional information

it works in edge and firefox
and it works in [ruffle-regressions.github.io](https://ruffle-regressions.github.io/) in chrome
is this a bug of chrome
![Image](https://github.com/user-attachments/assets/2c238995-30f3-409b-a362-6ea1ea805edf)
![Image](https://github.com/user-attachments/assets/4a6b9a2b-14b3-4c4d-8a6b-722eb4ee2e60)

**Comment by kjarosh** [comment]
Could you paste here the debug info from the context menu?

**Comment by zjk261** [comment]
```

# Player Info
Allows script access: false
Renderer: WebGL 2.0
Adapter Vendor: WebKit
Adapter Renderer: WebKit WebGL
Adapter Version: WebGL 2.0 (OpenGL ES 3.0 Chromium)
Surface samples: 4 x 
Surface size: 1440 x 691

# Page Info
Page URL: https://ruffle.rs/demo/

# Browser Info
User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/130.0.0.0 Safari/537.36
Platform: Win32
Has touch support: false

# Ruffle Info
Version: 0.1.0
Name: nightly 2024-10-19
Channel: nightly
Built: 2024-10-19T00:06:21.305Z
Commit: c39e8ada9ab160424766a0c87defb9cbeec42e93
Is extension: false

# Metadata
width: 800
height: 600
frameRate: 30
numFrames: 2
swfVersion: 10
backgroundColor: #000000
isActionScript3: true
uncompressedLength: 5107139
```
@kjarosh 
ok so i think this is because I set Preferred renderer to WebGL in extension and forgot to reset it to Automatic

**Comment by kjarosh** [comment]
Yes, I believe filters are not implemented for the WebGL (non-wgpu) backend.

**Comment by SlavaRashevskiyOffic** [comment]
I just noticed! When you play Blosics 2 Level Pack, then the 2 number is larger than should be.
